embeddings/task/lightning_task/sequence_labeling.py

```diff
@@ -1,138 +1,60 @@
-from collections import ChainMap
-from typing import Any, Dict, List, Optional, Tuple
+from typing import Any, Dict, List
 
 import numpy as np
-import torch
 from numpy import typing as nptyping
-from pytorch_lightning.utilities.types import STEP_OUTPUT
 from torch.utils.data import DataLoader
-from torchmetrics import F1, Accuracy, MetricCollection, Precision, Recall
-from transformers import AutoModelForTokenClassification
 
-from embeddings.data.datamodule import HuggingFaceDataset
-from embeddings.task.lightning_task.lightning_task import HuggingFaceLightningTask
-from embeddings.utils.loggers import get_logger
-from embeddings.utils.utils import NDArrayInt
+from embeddings.data.io import T_path
+from embeddings.model.lightning_module.sequence_labeling import SequenceLabelingModule
+from embeddings.task.lightning_task.lightning_task import LightningTask
 
-_logger = get_logger(__name__)
-
-
-class SequenceLabeling(HuggingFaceLightningTask):
-    downstream_model_type = AutoModelForTokenClassification
 
+class SequenceLabelingTask(LightningTask):
     def __init__(
         self,
-        model_name_or_path: str,
+        embedding_name: str,
```

Comment:
I would standardize the naming of arguments. What do you think about changing it to  `embedding_name_or_path` or `model_name_or_path`? Please check other classes in case of refactoring.